packages/infra/config/env-config.ts

```diff
@@ -37,6 +37,9 @@ export type EnvConfig = {
   analyticsSecretNames?: {
     POST_HOG_API_KEY: string;
   };
+  carequality?: {
+    CQ_API_KEY: string;
```

Comment:
Shouldn't we have code on CDK to expose this as env var?

Also, I suggest we leave all caps to env vars, this here is just a property that will be used to populate an env var.